testutil/network/network.go

```diff
@@ -8,13 +8,14 @@ import (
 	"github.com/cosmos/cosmos-sdk/baseapp"
 	"github.com/cosmos/cosmos-sdk/crypto/hd"
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	pruningtypes "github.com/cosmos/cosmos-sdk/pruning/types"
 	servertypes "github.com/cosmos/cosmos-sdk/server/types"
 	"github.com/cosmos/cosmos-sdk/simapp"
-	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	"github.com/cosmos/cosmos-sdk/testutil/network"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/ignite/cli/ignite/pkg/cosmoscmd"
+	"github.com/jaekwon/testify/require"
```

Comment:
Not sure if this was intended or not, but since this looks pretty abandoned I would perhaps suggest using `github.com/stretchr/testify` instead.

Reply:
That came in with the updated ignite cli code.... let's see if the alternative works.

Reply:
my first inclination was to remove ignite altogether.

Reply:
we removed Ignite CLI - was causing problems in CI.

Reply:
> we removed Ignite CLI - was causing problems in CI.
Before remove it: 
- How to reproduce the problems?
- Should be fixed in github.com/ignite/cli v0.24.0

I have opened a new branch here
https://github.com/BitCannaGlobal/bcna/compare/main...SDK-v0.46.1-%26-IBC/go-v5-on-the-way
- ICA integration fixed
- IBC production version 5

Please fell free to collaborate there 🙏